products/healthcare/api.yaml

```diff
@@ -304,6 +304,61 @@ objects:
         description: |
           The fully qualified name of this dataset
         output: true
+      - !ruby/object:Api::Type::Array
+        name: streamConfigs
+        description: |-
+          A list of streaming configs that configure the destinations of streaming export for every resource mutation in
+          this FHIR store. Each store is allowed to have up to 10 streaming configs. After a new config is added, the next
+          resource mutation is streamed to the new location in addition to the existing ones. When a location is removed
+          from the list, the server stops streaming to that location. Before adding a new config, you must add the required
+          bigquery.dataEditor role to your project's Cloud Healthcare Service Agent service account. Some lag (typically on
+          the order of dozens of seconds) is expected before the results show up in the streaming destination.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Array
+              name: 'resourceTypes'
+              description: |
+                Supply a FHIR resource type (such as "Patient" or "Observation"). See 
+                https://www.hl7.org/fhir/valueset-resource-types.html for a list of all FHIR resource types. The server treats
+                an empty list as an intent to stream all the supported resource types in this FHIR store.
+              item_type: Api::Type::String
+            - !ruby/object:Api::Type::NestedObject
+              name: bigqueryDestination
+              required: true
+              description: |
+                The destination BigQuery structure that contains both the dataset location and corresponding schema config.
+                The output is organized in one table per resource type. The server reuses the existing tables (if any) that
+                are named after the resource types, e.g. "Patient", "Observation". When there is no existing table for a given
+                resource type, the server attempts to create one.
+                See the [streaming config reference](https://cloud.google.com/healthcare/docs/reference/rest/v1beta1/projects.locations.datasets.fhirStores#streamconfig) for more details.
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: datasetUri
+                  required: true
+                  description: |
+                    BigQuery URI to a dataset, up to 2000 characters long, in the format bq://projectId.bqDatasetId
+                - !ruby/object:Api::Type::NestedObject
+                  name: schemaConfig
+                  required: true
+                  description: |
+                    The configuration for the exported BigQuery schema.
+                  properties:
```

Comment:
Would it make sense to mark one of these properties as `required` (probably schemaType?)? Just to prevent the empty block situation here: https://github.com/terraform-providers/terraform-provider-google/wiki/Developer-Best-Practices#default-values

Reply:
What is your opinion on `schemaType`? I thought about just removing it since there was only one valid option, and it feels weird requiring it. I agree we should avoid the empty block however

Reply:
Well if we remove it then we have to make the other field required, right?

I guess if we end up marking the recursive field as required then we could keep this one around with a default value

I don't have strong opinions either way

Reply:
Yep, making `recursiveStructureDepth` required seems fine, I'll do that.